common/config/environment.go

```diff
@@ -29,6 +29,7 @@ const (
 	DDInfraDeployFakeintakeWithLoadBalancer = "deployFakeintakeWithLoadBalancer"
 	DDInfraOSAmiID                          = "osAmiId"
 	DDInfraExtraResourcesTags               = "extraResourcesTags"
+	DDInfraContainerRuntime                 = "containerRuntime"
```

Comment:
Put in common configuration as it's possible that GKE and EKS add support for kata. Minikube also supports it. It can also be ran without kubernetes.

Reply:
how do you see the value? for example `containerd_runc` vs `containerd_kata` ? or should we have 2 variables: `ContainerRuntime` and `ContainerRuntimeFlavor` ?

Reply:
I think those two should be seperate concerns. This field could potentially be used outside of kubernetes, if we ever decide to implement some e2e tests using the containerd/docker socket.